channels/channel.mtg/viafree/chn_viafree.py

```diff
@@ -495,10 +496,8 @@ def create_video_item(self, result_set):
 
             # if the name had the episode in it, translate it
             if episode > 0 and season > 0:  # and not webisode:
-                description = "%s\n\n%s" % (title, description)
-                title = "{0} - s{1:02d}e{2:02d}".format(result_set["format_title"],
-                                                        season,
-                                                        episode)
+                is_episodic = True
```

Comment:
Remove this `is_episodic` and if you define season and episode as None or 0 inbetween line 488 and 489 you can check those later.

Reply:
Deletes line 499: "is_episodic = True"
Changes line 467 "is_episodic = False" to:
season = 0
episode = 0